docs/dev/reference/glossary/flashed.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Flashed
```

Comment:
Shouldn't this just be "flash"?

Reply:
this was what I specified. We can change to flash though